src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -107,6 +107,11 @@ export class QueryRunner extends Component<
         sampleUrl,
         selectedVersion: queryVersion
       });
+      telemetry.trackEvent(DROPDOWN_CHANGE_EVENT, 
+       { 
+         ComponentName: 'Version Change Dropdown', 
+         SelectedVersion: queryVersion 
```

Comment:
there's only two options but but we may want to consider capturing the old value and new value.